test/functional/p2p_post_invalidate_sync.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#
```

Comment:
Prefer to use file docstrings over code comments for the top-level comment

Reply:
I don't know what a docstrng is.

Reply:
Here you go: https://www.python.org/dev/peps/pep-0257/